src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/28.diff.sql

```diff
@@ -413,6 +413,8 @@ BEGIN
     )
     WITH (DROP_EXISTING=ON, ONLINE=ON)
 
+    DROP INDEX IF EXISTS IX_Instance_PartitionKey_Status_StudyInstanceUid_SeriesInstanceUid_SopInstanceUid on dbo.Instance
```

Comment:
Don't we also need to drop the old indexes?

Reply:
The problem is these indexes are already created and it failed on the UNIQUE index  creation.